scripts/workers/code_upload_worker_utils/install_dependencies.sh

```diff
@@ -31,6 +31,12 @@ echo "### Kubectl Installed"
 curl https://raw.githubusercontent.com/aws-samples/amazon-cloudwatch-container-insights/latest/k8s-deployment-manifest-templates/deployment-mode/daemonset/container-insights-monitoring/quickstart/cwagent-fluentd-quickstart.yaml | sed "s/{{cluster_name}}/$CLUSTER_NAME/;s/{{region_name}}/$AWS_DEFAULT_REGION/" | kubectl apply -f -
 echo "### Container Insights Installed"
 
+# Setup EFS as persistent volume
```

Comment:
At the moment, this will get set up for every cluster, right?

Reply:
We should add a check in the database for a challenge phase if it requires the persistent volume or not.

Reply:
Makes sense. I will add that check

Reply:
But this will complicate things in bash script. We have to somehow pass a list of tuples Efs id, pvc name, pv name and use some logic in code upload worker to get mapping of which pvc and pv to mount on which pod